### DISCUSSION TITLE
Include PII for LGPD (Lei Geral de Proteção de Dados)

### DISCUSSION THREAD

**Discussion by felipecasali-usp** [microsoft/presidio#1161] [Ideas]
Is the any way to add the PII fields that are in the scope of LGPD (Lei Geral de Proteção de Dados)?

**Comment by omri374** [comment]
Hi, I'm not familiar with LGPD, but there are multiple ways to add support for new PII:
- https://microsoft.github.io/presidio/analyzer/adding_recognizers/

As an OSS we cannot guarantee any compliance with GDPR, LGPD etc., and it's up to the user of the tool to evaluate the compliance aspects. 

If you're interested in contributing new recognizers for Brazilian PII, we'd be more than happy to review and include those as predefined ones.
